docs/source/conf.py

```diff
@@ -19,6 +19,8 @@
 import sys
 sys.path.insert(0, os.path.abspath('../../'))
 
+from maestrowf import __version__, __version_info__
```

Comment:
It looks likes this line needs to be moved up past the first `sys.path` call -- `flake8` doesn't like where this is.

Reply:
Fixed.